### PR TITLE
Added the licence block of readme

### PR DESCRIPTION
**Pull request by TheDarkMikeRises**
It now follows the format of other Microsoft repositories.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -137,3 +137,6 @@ provided by the bot. You will only need to do this once across all repos using o
 This project has adopted the [Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct/).
 For more information see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeofconduct/faq/) or
 contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additional questions or comments.
+
+## Licence
```
Comment (vdice) [inline README.md]: typo here: `License`, otherwise LGTM and thank you!

**Comment by radu-matei** [comment]
Ping, @TheDarkMikeRises 
Also, why two commits? What does the second one change?

Thanks!

**Comment by technosophos** [comment]
Any update on this?

**Comment by radu-matei** [comment]
Closing, as there is no response on this - but feel free to re-open if you want to pick it up again, we'd love to help.

Thanks!

**Pull request closed without merging**